docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,172 @@
+Afterwards, the images for the Garden cluster are built and deployed into the KinD cluster.
+Finally, the images for the Seed cluster are built, pushed to a container registry on the Seed and the `gardenlet` is started.
+
+## Pause and unpause the KinD cluster
+
+Unlike in the `getting started locally` scenario the KinD cluster can be paused by stopping and keeping its docker container. This is done by:
+
+```bash
+make kind-extensions-down
+```
+
+When you run `make kind-extensions-up` again, you will start the docker container with your previous Gardener configuration again. 
+
+This provides the option to switch off your local KinD cluster fast without leaving orphaned infrastructure elements behind.
+
+
+## Creating a `Shoot` cluster
+
+You can wait for the `Seed` to be ready by running
+
+```bash
+kubectl wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed provider-extensions --timeout=5m
+```
+
+Alternatively, you can run `kubectl get seed provider-extensions` and wait for the `STATUS` to indicate readiness:
+
+```bash
+NAME                  STATUS   PROVIDER   REGION         AGE    VERSION      K8S VERSION
+provider-extensions   Ready    gcp        europe-west1   111m   v0.0.0-dev   v1.24.2
+```
+
+In order to create a first shoot cluster, just try one of the following example shoot configurations. Please remember that only shoot cluster with the same provider type as your seed cluster will be started successfully. The others will remain in `<pending>` state.
+
+```bash
+kubectl apply -f ./example/provider-extensions/shoot/aws.yaml
+kubectl apply -f ./example/provider-extensions/shoot/aws-arm64.yaml
+kubectl apply -f ./example/provider-extensions/shoot/azure.yaml
+kubectl apply -f ./example/provider-extensions/shoot/gcp.yaml
+```
+
+You can wait for the `Shoot` to be ready by running `kubectl -n garden-local get shoots` and wait for the `LAST OPERATION` to reach `100%`:
+
+```bash
+NAME        CLOUDPROFILE   PROVIDER   REGION         K8S VERSION   HIBERNATION   LAST OPERATION               STATUS    AGE
+aws         aws            aws        eu-west-1      1.24.3        Awake         <pending>                    healthy   84s
+aws-arm64   aws            aws        eu-west-1      1.24.3        Awake         <pending>                    healthy   65s
+azure       az             azure      westeurope     1.24.2        Awake         <pending>                    healthy   57s
+gcp         gcp            gcp        europe-west1   1.24.3        Awake         Create Processing (43%)      healthy   94s
+```
+
+### Accessing the `Shoot` cluster
+
+Your shoot clusters will have a public DNS entries for their API servers, so that they are could be reached via `kubectl` after you downloaded their `kubeconfig`.
+
+The cluster from the examples listed before have kubeconfigs with static tokens. This is how to access the `gcp` cluster example. The others could be accessed accordingly. 
```

Comment:
Fix this, i.e. advertise using the `adminkubeconfig` subresource instead of the static kubeconfig (even if enabled).